File: Kaskus/CueText/Dede.App.CueTextCs/FormMain.cs
namespace Dede.App.CueText
{
    using System.Windows.Forms;

    public partial class FormMain 
        : Form
    {
        public FormMain()
        {
            this.InitializeComponent();

            this.textBoxCueTest.CueText(@"Cue TextBox");
            this.maskedTextBoxCueTest.CueText(@"Cue MaskedTextBox");
            this.comboBoxCueTest.CueText(@"Cue ComboBox");
        }
    }
}

File: Kaskus/CueText/CueTextVb/FormMain.vb
Imports Dede.App.CueText.Modules

Public Class FormMain
    Sub New()
        InitializeComponent()

        textBoxCueTest.CueText("Cue TextBox")
        maskedTextBoxCueTest.CueText("Cue MaskedTextBox")
        comboBoxCueTest.CueText("Cue ComboBox")
    End Sub
End Class